iocage_lib/ioc_fstab.py

```diff
@@ -35,9 +36,36 @@
 import iocage_lib.ioc_exceptions
 import texttable
 import ctypes
-from ctypes.util import find_library
 from collections import OrderedDict
 
+from iocage_lib.utils import load_ctypes_library, ensure_unicode_str
+
+
+class Fstab(ctypes.Structure):
+    _fields_ = [
+        ('fs_spec', ctypes.c_char_p),
+        ('fs_file', ctypes.c_char_p),
+        ('fs_vfstype', ctypes.c_char_p),
+        ('fs_mntops', ctypes.c_char_p),
+        ('fs_type', ctypes.c_char_p),
+        ('fs_freq', ctypes.c_int),
+        ('fs_passno', ctypes.c_int),
+    ]
+
+
+fstab_pointer = ctypes.POINTER(Fstab)
+
+FSTAB_SIGNATURES = {
+    'setfstab': ([ctypes.c_char_p], ctypes.c_int),
+    'getfstab': ([], ctypes.c_char_p),
+    'getfsent': ([], fstab_pointer),
+    'endfsent': ([], None),
+}
```

Comment:
If this isn ot used anywhere else it sohuld be lowercase or just argument to load_ctypes_library